docs/source/tutorial.rst

```diff
@@ -24,4 +24,5 @@ Jupyter Tutorials
     tutorials/barren_plateaus.ipynb
     tutorials/qaoa_portfolio_optimization.ipynb
     tutorials/imag_time_evo.ipynb
-    tutorials/qcloud_sdk_demo.ipynb
\ No newline at end of file
+    tutorials/qcloud_sdk_demo.ipynb
+    tutorials/sklearn_svc.ipynb
```

Comment:
please move this one row higher: i.e. keep qcloud demo as the last tutorial

Reply:
fixed!